compose/cli/main.py

```diff
@@ -328,13 +328,27 @@ def logs(self, project, options):
         Usage: logs [options] [SERVICE...]
 
         Options:
-            --no-color  Produce monochrome output.
+            --no-color          Produce monochrome output.
+            -f, --follow        Follow log output.
+            -t, --timestamps    Show timestamps.
+            --tail="all"        Number of lines to show from the end of the logs
+                                for each container.
         """
         containers = project.containers(service_names=options['SERVICE'], stopped=True)
 
         monochrome = options['--no-color']
+        log_args = {}
+        log_args['follow'] = options['--follow']
+        log_args['timestamps'] = options['--timestamps']
```

Comment:
Minor thing, but this could be a single assignment with a dictionary literal.

```python
log_args = {
    'follow': options['--follow'],
    'timestamps': options['--timestamps'],
}
```